src/routes/get_media.rs

```diff
@@ -1,14 +1,14 @@
 use std::path::PathBuf;
+use rocket::{Rocket, State};
+use rocket_jwt::{JWT, RocketJWT};
 
-#[get("/get-media")]
-async fn get_media(manga_id: &str) {
-    //TODO: Check if Client is Authenticated
-    //TODO: Implement file server to serve manga file
+#[get("/manga/<id>")]
```

Comment:
can we change th endpoints "/manga/<id>" and "/mangas/" to "/media/<id>" and "/media-list/"?

Reply:
sure